WcaOnRails/app/models/competition.rb

```diff
@@ -26,6 +26,22 @@ class Competition < ActiveRecord::Base
   validates :venue, format: { with: PATTERN_TEXT_WITH_LINKS_RE }
   validates :website, format: { with: /\Ahttps?:\/\/.*\z/ }, allow_blank: true
 
+  def self.NEARBY_DISTANCE_KM_WARNING
```

Comment:
These are class methods, you can define and use constants like so and save the method invocation overhead:

```ruby
class Foo
  NEARBY_DISTANCE_KM_DANGER = 200

  def some_calculation
    NEARBY_DISTANCE_KM_DANGER * 10
  end
end

puts Foo:: NEARBY_DISTANCE_KM_DANGER
```